cmd/notation/inspect.go

```diff
@@ -56,27 +58,35 @@ func inspectCommand(opts *inspectOpts) *cobra.Command {
 	if opts == nil {
 		opts = &inspectOpts{}
 	}
-	command := &cobra.Command{
-		Use:   "inspect [reference]",
-		Short: "Inspect all signatures associated with the signed artifact",
-		Long: `Inspect all signatures associated with the signed artifact.
+	longMessage := `Inspect all signatures associated with the signed artifact.
 
 Example - Inspect signatures on an OCI artifact identified by a digest:
-  notation inspect <registry>/<repository>@<digest>
+	notation inspect <registry>/<repository>@<digest>
 
 Example - Inspect signatures on an OCI artifact identified by a tag  (Notation will resolve tag to digest):
-  notation inspect <registry>/<repository>:<tag>
+	notation inspect <registry>/<repository>:<tag>
 
 Example - Inspect signatures on an OCI artifact identified by a digest and output as json:
-  notation inspect --output json <registry>/<repository>@<digest>
-`,
+	notation inspect --output json <registry>/<repository>@<digest>
+`
+	experimentalExamples := `
+Example - [Experimental] Inspect signatures on an OCI artifact identified by a digest using the Referrers API, if not supported, fallback to the Referrers tag schema
+	notation inspect --allow-referrers-api <registry>/<repository>@<digest>
```

Comment:
Example indentation should not be changed.

Reply:
Oh yeah, nice catch. My VS Code is using tab as default.